joblib/__init__.py

```diff
@@ -129,7 +138,7 @@
 
 __all__ = ['Memory', 'MemorizedResult', 'PrintTime', 'Logger', 'hash', 'dump',
            'load', 'Parallel', 'delayed', 'cpu_count', 'effective_n_jobs',
-           'register_parallel_backend', 'parallel_backend',
+           'register_parallel_backend', 'parallel_backend', 'expires_after',
            'register_store_backend', 'register_compressor',
            'wrap_non_picklable_objects']
```

Comment:
Just Julien being nitpicky: eventually, this can be sorted alphabetically and formatted _à la_ `black` so as to have semantic changes be on a single line in the future, making the future changes easier to understand.